roofit/roofitcore/inc/RooAICRegistry.h

```diff
@@ -37,6 +37,7 @@ class RooAICRegistry {
   const std::vector<Int_t>&  retrieve(Int_t masterCode, pRooArgSet& set1, pRooArgSet& set2) const ;
   const std::vector<Int_t>&  retrieve(Int_t masterCode, pRooArgSet& set1,
                                       pRooArgSet& set2, pRooArgSet& set3, pRooArgSet& set4) const ;
+  size_t getSize() const;
```

Comment:
```suggestion
  size_t size() const;
```
We try to use the `size()` name as in the STL nowadays, especially if the return type is `size_t`.